doc/pagerank.adoc

```diff
@@ -31,26 +34,33 @@ CALL algo.pageRank(label:String, relationship:String, {iterations:5, dampingFact
 | name | type | default | optional | description
 | label  | string | null | yes | label to load from the graph, if null load all nodes
 | relationship | string | null | yes | relationship-type to load from the graph, if null load all nodes
-| iterations | int | 5 | yes | how many iterations of page-rank to run
+| iterations | int | 20 | yes | how many iterations of page-rank to run
 | dampingFactor | float | 0.85 | yes | damping factor of the page-rank caculation
 | write | boolean | true | yes | if result should be written back as node property
-| scoreProperty | string | 'score' | yes | property name written back to
+| writeProperty | string | 'score' | yes | property name written back to
+| scoreProperty | string | 'score' | yes | property name that contains weight
```

Comment:
Yes I think that it should be different defaults... i went with what I read from the code... I think that "scoreProperty" is "weightProperty" not sure... we should avoid using propertyName as it is confusing imo